llvm/lib/Target/AIE/AIECombinerHelper.cpp

```diff
@@ -901,16 +908,43 @@ bool modifyToS20(InstrNode Start, MachineRegisterInfo &MRI, MachineIRBuilder &B,
     Helper.tryCombineCopy(*StartNodeMI);
     return true;
   }
+  case TargetOpcode::G_INTRINSIC: {
+    if (TII->isS20Intrinsic(*StartNodeMI, MRI)) {
```

Comment:
Here we insert a generic vector extract, regardless of what the intrinsic is, so I think `isS20Intrinsic` is not an accurate name and should rather be `isVExtractIntrinsic`. Still, I'd rather have generic opcodes like mentioned in https://github.com/Xilinx/llvm-aie/pull/233/commits/d5d7cf087b0a27d02cdb7a0bf1f68f3e618470a1#r1839776194 and not have to deal with intrinsics at all.